digital/index.html

```diff
@@ -77,21 +75,32 @@ <h3 class="card-title">Reliable and maintainable services</h3>
   </div>
 </div>
 
-<div class="container">
-  <div class="row my-3">
-    <div class="col-md-6">
-      <img src="images/delivery.jpg" alt="Whiteboard from agile restrospective" class="DeliveryImage">
-    </div>
-    <div class="col-md-6">
-      <h5 class="card-title">Delivery</h5>
-      <p>
-        Delivery as a discipline is to deliver outcomes while considering the wellbeing and organisation of the team.
-      </p>
-      <p>
-        To ensure the needs of users are met, continuous delivery is used. Continuous delivery is a practice which we use to provide value as quickly and efficiently as possible.
-      </p>
+  <div class="container ven-diagram-section">
+    <div class="row">
+      <div class="col text-center ven-diagram">
+          <img src="images/venn-diagram.png" class="ven-diagram">
+      </div>
     </div>
   </div>
+
+<main>
+  <div class="container">
+    <div class="row my-3">
+      <div class="col-md-6">
+        <img src="images/delivery.jpg" alt="" style="width: 500px; height: 300px">
```

Comment:
This image is not going to scale on mobile, I suggest making a CSS class that specifies the `width` only and a `max-width` of 100%